docs/data/material/discover-more/backers/backers.md

```diff
@@ -93,32 +89,65 @@ Funds transferred to the MUI for-profit support the company's mission.
 
 <span class="only-light-mode">
-  <img src="/static/readme/netlify-lightmode.svg" alt="Netlify logo" loading="lazy" width="100" height="27" style="margin-top: 1rem;">
+  <img src="/static/readme/netlify-lightmode.svg" alt="Netlify logo" loading="lazy" width="180" height="49" style="width: 100px; margin-top: 1rem;">
 </span>
 <span class="only-dark-mode">
-  <img src="/static/readme/netlify-darkmode.svg" alt="Netlify logo" loading="lazy" width="100" height="27" style="margin-top: 1rem;">
+  <img src="/static/readme/netlify-darkmode.svg" alt="Netlify logo" loading="lazy" width="180" height="49" style="width: 100px; margin-top: 1rem;">
 </span>
 
 [Netlify](https://www.netlify.com/) lets us distribute the documentation.
 
 <span class="only-light-mode">
-  <img src="/static/readme/browserstack-lightmode.svg" alt="BrowserStack logo" loading="lazy" width="140" height="25" style="margin-top: 1rem;">
+  <img src="/static/readme/browserstack-lightmode.svg" alt="BrowserStack logo" loading="lazy" width="180" height="32" style="width: 140px; margin-top: 1rem;">
 </span>
 <span class="only-dark-mode">
-  <img src="/static/readme/browserstack-darkmode.svg" alt="BrowserStack logo" loading="lazy" width="140" height="25" style="margin-top: 1rem;">
+  <img src="/static/readme/browserstack-darkmode.svg" alt="BrowserStack logo" loading="lazy" width="180" height="32" style="width: 140px; margin-top: 1rem;">
 </span>
 
 [BrowserStack](https://www.browserstack.com/) lets us test in real browsers.
 
-<img loading="lazy" alt="CodeCov logo" src="https://avatars.githubusercontent.com/u/8226205?s=70" width="35" height="35" style="margin-top: 1rem;">
+<img loading="lazy" alt="CodeCov logo" src="https://avatars.githubusercontent.com/u/8226205?s=70" width="70" height="70" style="width: 35px; margin-top: 1rem;">
 
 [CodeCov](https://about.codecov.io/) lets us monitor test coverage.
+
+## Tier benefits
+
+### Diamond
+
+Your name/company logo will be placed in:
+
+- A prominent logo placement in sidebar of all content pages (4m+ sessions, 900k+ unique visitors per month)
+- ⬇️ same as the lower tiers
+
+Please contact us at sales@mui.com before subscribing to this tier to get preliminary approval.
+
+### Gold
+
+Your name/company logo will be placed in:
+
+- the [homepage](https://mui.com/#sponsors) (500k+ pageviews and 150k+ unique visitors per month)
+- the [README.md](https://github.com/mui/material-ui#sponsors) (80k+ unique visitors per month)
+- the [sponsors' list](#gold-sponsors)
+
+\*we accept donations, but we won't show unethical ads.
+
+### Silver
+
+Your name/company logo will be placed in the [sponsors' list](#silver-sponsors).
+
+### Bronze
+
+Your name/company logo will be placed in the [sponsors' list](#bronze-sponsors).
+
+### Backers
+
+Help support office hours and further development by becoming a Backer. We'll thank you by including your avatar in the [backers list](#backers).
```

Comment:
This is the first I'm hearing about "office hours." 😅 Maybe we should add more details here about what that would be and why it would be appealing to a potential backer—sell them on it!

Reply:
Right, it's not clear, we can do this instead:

```suggestion
Help support more open-source development by becoming a Backer. We'll thank you by including your avatar in the [backers list](#backers).
```